### PR TITLE
[KOGITO-1600] - Prepare 0.9.0 Kogito Operator release

### PR DESCRIPTION
**Pull request by ricardozanini**
Signed-off-by: Ricardo Zanini <zanini@redhat.com>

See:
https://issues.redhat.com/browse/KOGITO-1600

Nitpicks/scorecard check for 0.9.0 release.

Many thanks for submiting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](CONTRIBUTING.MD#sending-a-pull-request)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [x] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -136,6 +136,7 @@ run-tests:
 		--examples_uri ${examples_uri} \
 		--examples_ref ${examples_ref} \
 		--namespace_name ${namespace_name} \
+		--load_factor ${load_factor} \
```
Comment (spolti) [inline Makefile]: what is this load factor?

Comment (sutaakar) [reply Makefile]: See https://issues.redhat.com/browse/KOGITO-1338

**Comment by ricardozanini** [comment]
Waiting for https://issues.redhat.com/browse/KOGITO-1601

**Comment by ricardozanini** [comment]
OperatorHub PR: https://github.com/operator-framework/community-operators/pull/1475

**Pull request merged**